components/MapsDedi/index.vue

```diff
@@ -59,11 +141,230 @@ export default {
     isMitra () {
       return this.joinSelected === 'Mitra'
     }
+  },
+  mounted () {
+    this.initialGoogleMap()
+  },
+  methods: {
+    /**
+     * Initial load google map
+     */
+    async initialGoogleMap () {
+      try {
+        const google = await loader.load()
+        const map = new google.maps.Map(this.$refs.maps, mapOptions)
+
+        this.map = map
+
+        map.addListener('click', () => {
+          if (this.infoWindow) {
+            this.infoWindow.close()
+          }
+        })
+
+        map.addListener('dragend', () => {
+          if (this.infoWindow) {
+            if (!this.inBounds(this.infoWindow.position, this.getBounds(map))) {
+              this.infoWindow.close()
+            }
+          }
+        })
+
+        this.setMarker(google, map)
+      } catch (error) {
+        throw new Error(error)
```

Comment:
discussion:

I think I've read Adrian's comment about `throw new Error()` on a catch block on @naufalihsank PR a while ago, but I could be wrong 🤔

Reply:
what did he say? and what the issues ?